fhir-config/src/main/java/org/linuxforhealth/fhir/config/FHIRConfigHelper.java

```diff
@@ -104,8 +104,8 @@ private static JsonValue getPropertyFromTenantOrDefault(String propertyName) {
     }
 
     /**
-     * This generic function will perform the work of retrieving a property from either the tenant-specific config, or
-     * the default config, and then converting the resulting value to the appropriate type.
+     * This generic function will perform the work of retrieving a property from either the tenant-specific
+     * config or the default config, and then converting the resulting value to the appropriate type.
```

Comment:
just some style changes to reduce line lengths in this file...not related to rest of PR